### PR TITLE
Static access to LOOPBACK_INTERFACE

### PR DESCRIPTION
**Pull request by gpotter2**
Current code wont work on Npcap.
I didn't manage to write some convenient tests, so I added a warning

### CODE REVIEW

File: scapy/route.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 from __future__ import absolute_import
 
-
+import scapy.consts
 from scapy.config import conf
-from scapy.consts import WINDOWS, LOOPBACK_INTERFACE
+from scapy.consts import WINDOWS
```
Comment (p-l-) [inline scapy/route.py]: Could you remove this line and use `scapy.consts.WINDOWS` instead of `WINDOWS`?

Comment (gpotter2) [reply scapy/route.py]: Well WINDOWS is guessed on loading only from inside const.py... Is it really necessary ?
Such code is used many times in scapy

Comment (p-l-) [reply scapy/route.py]: Importing `scapy.consts` **and** `WINDOWS` from `scapy.consts` makes loading this module load `scapy.consts` twice.

Comment (p-l-) [reply scapy/route.py]: What you could do maybe is using `from scapy import consts` and use `consts.WINDOWS` and `consts.LOOPBACK_INTERFACE` if you prefer?

Comment (gpotter2) [reply scapy/route.py]: Fixed !

**Pull request merged**